src/controllers/expenses.js

```diff
@@ -0,0 +1,147 @@
+'use strict';
+
+const expenseServise = require('../services/expenses.js');
+
+const userServise = require('../services/users');
+
+const getAllExpenses = (req, res) => {
+  const expenses = expenseServise.getAllExpenses();
+  const { userId, category, from, to } = req.query;
+
+  if (category) {
+    const foundExpenses = expenseServise.filter(
+      (expense) => expense.category === category
+    );
+
+    res.statusCode = 200;
+
+    res.send(foundExpenses);
+
+    return;
+  }
+
+  if (userId) {
+    const foundExpenses = expenseServise.filter(
+      (expense) => expense.userId === Number(userId)
+    );
+
+    res.statusCode = 200;
+
+    res.send(foundExpenses);
+
+    return;
+  }
+
+  if (!expenses.length) {
+    res.send([]);
+
+    return;
+  }
+
+  if (from && to) {
+    const foundExpenses = expenseServise.filter(
+      (expense) => expense.spentAt >= from && expense.spentAt <= to
+    );
+
+    res.statusCode = 200;
+
+    res.send(foundExpenses);
+
+    return;
+  }
+
+  res.statusCode = 200;
+
+  res.send(expenseServise.getAllExpenses());
+};
+
+const addExpense = (req, res) => {
+  const { userId } = req.body;
+
+  const foundUser = userServise.getUserById(userId);
+
+  if (!foundUser) {
+    res.statusCode = 400;
+
+    res.send();
+
+    return;
```

Comment:
Why you can't just `return` in this case? This way 👇 
```suggestion
    return;
```